docs/imagecustomizer/api/configuration/verity.md

```diff
@@ -39,13 +38,25 @@ storage:
     - id: var
       size: 2G
 
+    - id: usr
```

Comment:
Let's have two separate examples: One for / and one for /usr. While you can technically use them together, most of the time a user will use one or the other.

Also, the /usr example can probably be a little simpler since you don't need to split of /var into its own partition.